receiver/mongodbatlasreceiver/mongodb_event_to_logdata.go

```diff
@@ -128,6 +128,7 @@ func mongodbAuditEventToLogData(logger *zap.Logger, logs []model.AuditLog, pc Pr
 
 		attrs.PutInt("result", int64(log.Result))
 
+		//nolint:errcheck
```

Comment:
Could you add a comment on the places you are ignoring the linter justifying why you are using it? I think it's fair to use it, but in general we should require a (brief) comment justifying the use

Reply:
I think we should fix this, the reason I added is to then followup soon to fix this and not to keep it there.